### PR TITLE
Fix the CI Errors Found in the GH Action

### LINKED ISSUE #438: Hotfix: Fix the Broken Things Found on the CI/CD Actions
There were a few errors found on the actions. Addressing the CI ones:

https://github.com/microsoft/vsts-extension-retrospectives/actions/runs/3101261381
and
https://github.com/microsoft/vsts-extension-retrospectives/actions/runs/3101261375

CD needs to be fixed in a different PR.

### PR DESCRIPTION
**Pull request by mindlessroman**
Fixes #438 for front end CI problems.

### CODE REVIEW

**Comment by mindlessroman** [comment]
FWIW, CD errors will be addressed separately.

File: RetrospectiveExtension.Frontend/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "@typescript-eslint/parser": "5.30.7",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.7",
     "autoprefixer": "10.4.7",
+    "cheerio": "^1.0.0-rc.3",
```
Comment (polatengin) [inline RetrospectiveExtension.Frontend/package.json:68]: I don't see we use "cheerio" in the project. is it added for future use?

**Review by polatengin — APPROVED** [review]
LGTM! thanks 👍

File: RetrospectiveExtension.Frontend/package.json (hunk 1 above)
Comment (mindlessroman) [reply RetrospectiveExtension.Frontend/package.json:68]: It's a related dependency from Jest, iirc from what I could gather based on some errors I was getting (https://github.com/microsoft/vsts-extension-retrospectives/actions/runs/3101588532/jobs/5023103595#step:5:301) - https://github.com/enzymejs/enzyme/issues/2518#issuecomment-1133447589

**Pull request merged**